src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -34,6 +35,16 @@ export const GAME_OPTION_VALUES = [
         ]
     },
     {
+        name: GAME_OPTION_NAMES.PTL_LENGTH,
+        defaultValue: '1',
+        description: 'Projected Track Line (PTL) Length',
+        type: 'select',
+        data: [
+            ['1', '1'],
+            ['0.5', '0.5']
```

Comment:
Yeah, I'm not able to reproduce a 2min PTL going that far. Looks like it takes more than a 10 minute to stretch that far in most cases. Anyway, I'm not suggesting that anyone would leave it on such an absurd setting, but giving it as an option... we might as well. It would be a pain to change back and forth from settings like this, but the plan is for vector lines to be mapped to a hotkey (decrease/increase with F2/F3; see #14), and that's when the longer lengths really become useful: when you can run them out real quick to show a projection between converging traffic, then a few taps to shrink them back down to normal size.